forte/data/readers/conll03_reader_new.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+from typing import Iterator, Any
+from ft.onto.base_ontology import Token, Sentence, Document, EntityMention
+from forte.data.data_pack import DataPack
+from forte.data.data_utils_io import dataset_path_iterator
+from forte.data.readers.base_reader import PackReader
+
+__all__ = [
+    "CoNLL03Reader"
+]
+
+
+class CoNLL03Reader(PackReader):
+    r""":class:`CoNLL03Reader` is designed to read in the CoNLL03-ner dataset.
```

Comment:
Can you document exactly what is a Conll03 ner dataset? Like download like, expected format, etc.

Reply:
Done.